face_api_dataset/dataset.py

```diff
@@ -2,12 +2,18 @@
 import json
 import os
 from collections import Sequence
-from enum import Enum, auto
+from enum import Enum
+from enum import auto
 from importlib.util import find_spec
 from pathlib import Path
 from typing import Optional, List, Any, Union, Dict, Callable, overload, TYPE_CHECKING, Tuple
 
 import numpy as np
+import pandas as pd
```

Comment:
If we use pandas it should be added to `setup.py`. Does it save a lot of time? If not then it's probably better not to use it to avoid unnecessary dependency.

Reply:
fixed